src/content/is-ghostery-browser.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+(async function () {
```

Comment:
why async? does it need one tick? maybe requestAnimationFrame would be faster?

Reply:
async was by mistake. Removed.